docs/en/cookbook/working-with-datetime.rst

```diff
@@ -58,25 +58,88 @@ Handling different Timezones with the DateTime Type
 
 If you first come across the requirement to save different timezones you may be still optimistic about how
 to manage this mess,
-however let me crush your expectations fast. There is not a single database out there (supported by Doctrine 2)
-that supports timezones correctly. Correctly here means that you can cover all the use-cases that
-can come up with timezones. If you don't believe me you should read up on `Storing DateTime
-in Databases <http://derickrethans.nl/storing-date-time-in-database.html>`_.
+however let me crush your expectations fast. Each database (that is supported by Doctrine 2) handles timezones in a different way - if they handle it at all! If you don't believe me, you can find more details on that in different blogposts about `the general aproach <http://derickrethans.nl/storing-date-time-in-database.html>`_ and - more up-to-date - about timezone-handling in `Postgres <https://andreas.heigl.org/2016/05/29/timezones-and-postgresql/>`_ and `MySQL <https://andreas.heigl.org/2016/04/18/timezones-and-mysql/>`_. 
 
 The problem is simple. Not a single database vendor saves the timezone, only the differences to UTC.
 However with frequent daylight saving and political timezone changes you can have a UTC offset that moves
 in different offset directions depending on the real location.
 
-The solution for this dilemma is simple. Don't use timezones with DateTime and Doctrine 2. However there is a workaround
-that even allows correct date-time handling with timezones:
+The solution for this dilemma seems simple. Don't use timezones with DateTime and Doctrine 2. However there are 2 workarounds
```

Comment:
> The solution for this dilemma seems simple. Don't use timezones with DateTime and Doctrine 2.

Maybe is better?:

The solution for this dilemma seems simple: don't use timezones with DateTime and Doctrine 2.